domain/sysvar_cache.go

```diff
@@ -201,6 +202,13 @@ func checkEnableServerGlobalVar(name, sVal string) {
 		variable.TopSQLVariable.ReportIntervalSeconds.Store(val)
 	case variable.TiDBRestrictedReadOnly:
 		variable.RestrictedReadOnly.Store(variable.TiDBOptOn(sVal))
+	case variable.TiDBStoreLimit:
+		var val int64
+		val, err = strconv.ParseInt(sVal, 10, 64)
+		if err != nil {
+			break
+		}
+		storekv.StoreLimit.Store(val)
```

Comment:
This switch for if you change a global var, and you want to run code on other TiDB servers. I just wanted to confirm this is the case here?

(I had thought we would not need to add items to this switch, but that's turned out to be wrong. We may need to change `SetGlobal()` so that it fires on all TiDB servers).

Reply:
Yes we need this. the case is that:

|  statement   | Node1 instance  | Node2 instance |
|  ----  | ----  | ---- |
| set global  | yes(apply local instance) | yes(propagated to instance2) |
| set session| yes(apply local instance) | no |

I investigate for a while, maybe this code is actually what I want.